sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/Appearance.java

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.ai.formrecognizer.models;
```

Comment:
U probably forget this is a explored model. Need another visited of this model class

Reply:
Also there are more explored models but still have public setter. I guess u are still working on it. I will take another look at this PR later.

Reply:
It is a generated model, keeping it as is for this preview.

Reply:
Since it is explored to user. We can make it not settable.

Reply:
Good point, updated. I was under the impression since this is mostly generated code as is shouldn't need to worry but I guess it will help to avoid breaking changes.